src/Microsoft.ML.PCA/PcaTransform.cs

```diff
@@ -722,64 +720,4 @@ public SchemaShape GetOutputSchema(SchemaShape inputSchema)
             return new SchemaShape(result.Values);
         }
     }
-
-    public static class PcaEstimatorExtensions
```

Comment:
>PcaEstimatorExtensions [](start = 24, length = 22)

Where did this code move? I don't find it elsewhere. Is it not needed?

Reply:
Moved to  `Microsoft.ML.StaticPipe\TransformsStatic.cs`

Line #1625

---
In reply to: [244831747](https://github.com/dotnet/machinelearning/pull/1930#discussion_r244831747) [](ancestors = 244831747)